AT_Training/Exercise_1/Program.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+
+namespace Exercise_1
+{
+    class Program
+    {
+        static void Main(string[] args)           
+        {
+            double X, Y, Z;
+            do
+            {
+                try
+                {
+                Console.Write("X:");
+                    X = Convert.ToDouble(Console.ReadLine());
+                    break;
+                }
+                catch (FormatException)
```

Comment:
try catch is good :) but could be simplified to TryParse. without while (true) that will make code easier to read
But its ok too